src/artemis.hpp

```diff
@@ -67,11 +67,13 @@ namespace prim {
 ARTEMIS_VARIABLE(dust.prim, density);
 ARTEMIS_VARIABLE(dust.prim, velocity);
 } // namespace prim
+ARTEMIS_VARIABLE(dust, stopping_time);
 } // namespace dust
 #undef ARTEMIS_VARIABLE
 
 // TaskCollection function pointer for operator split tasks
-using TaskCollectionFnPtr = TaskCollection (*)(Mesh *pm, const Real time, const Real dt);
+using TaskCollectionFnPtr = TaskCollection (*)(Mesh *pm, parthenon::SimTime &tm,
```

Comment:
This will break coupling to `jaybenne`, but I am happy to make the change.  If we are passing the whole `SimTime` object, however, should we still pass `dt`? Maybe we would want to pass a different `dt` than `tm.dt` for something like sub-cycling...

Reply:
I need SimTime object. It would be better to keep both.

Reply:
@pdmullen Can subcycling not always be done inside the enrolled task collection? It seems to me to make sense to always provide the `SimTime` argument here, but maybe I'm missing something.

Reply:
I would prefer that anything that returns a `TaskCollection` lives inside the driver class. You then have access to everything in the driver class like `SimTime`.